wiki/en/Running-a-Server.md

```diff
@@ -77,18 +64,20 @@ This is the default when starting a Server for the first time. Unregistered Serv
 	<figcaption>How Unregistered Servers work</figcaption>
 </figure>
 
+If you are running an unregistered server behind a home internet connection, you might need to enable [port forwarding](#port-forwarding) as described below.
+
 ### 3. Custom Directory
 
 If you want to run a number of Servers, possibly also behind a firewall or on a LAN, you may want to run your Server as a Directory. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
 
-To run a Custom Directory [read this guide](Custom-Directories)
+To run a Directory [read this guide](Custom-Directories)
 
 
-# Installing and running a Server 
+## Installing and running a Server
```

Comment:
The whole page is "Running a Server".  This should focus on "Installation".

Reply:
Done.